profiles/VP_ANDROID_15.json

```diff
@@ -0,0 +1,205 @@
+{
+    "$schema": "https://schema.khronos.org/vulkan/profiles-0.8.1-237.json#",
```

Comment:
Many features in this profile requires a newer schema than 237. Once, I removed the unknown extension, I figure the minimum schema version is 266, https://schema.khronos.org/vulkan/profiles-0.8.1-266.json because `VK_ANDROID_external_format_resolve` was added with that version.

Reply:
Fixed, thanks!

Reply:
So it looks like this would need to be updated once the extension MRs are merged.

Reply:
Are we good now, or does this need to be updated again?